Applications/Gorm/GormAppDelegate.m

```diff
@@ -33,6 +33,28 @@
 #import "GormAppDelegate.h"
 #import "GormLanguageViewController.h"
 
+@interface Gorm : NSApplication
+@end
+@implementation Gorm
+
+/*
+   NSApplication override to make Inspector's shortcuts available globally
+*/
+- (void) sendEvent: (NSEvent *)theEvent
```

Comment:
This change concerns me because it is not the default behavior in other applications.   Why do the key equivalents of every inspector need to be handled GLOBALLY?   Can you explain what bug this fixes for you, what is the use-case you're attempting to address?

Reply:
The other issue here is that there was a reason I moved AWAY from making Gorm use an NSApplication subclass.

Reply:
> This change concerns me because it is not the default behavior in other applications.

Any application can be subclassed from NSApplication and redefine default shortcuts handling. I see now problem here.

> Why do the key equivalents of every inspector need to be handled GLOBALLY? Can you explain what bug this fixes for you, what is the use-case you're attempting to address?

There's no bug it's usability enhancement. For example, if user wants to see what connections selected object has (whithout focus switch) it's handy to press shortcut whithout moving mouse and selecting "Connections" sections of Inpector.

Reply:
> The other issue here is that there was a reason I moved AWAY from making Gorm use an NSApplication subclass.

Could you please explain that reason? It might be useful for development practice.